src/hb-buffer.h

```diff
@@ -112,15 +111,26 @@ typedef struct hb_glyph_info_t {
  * 				   reshaping to a small piece around the
  * 				   breaking point only, even if the breaking
  * 				   point carries the
- * 				   #HB_GLYPH_FLAG_UNSAFE_TOBREAK, by reshaping
- * 				   a the shortest segment from the closest
- * 				   safe-to-concat point.
+ * 				   #HB_GLYPH_FLAG_UNSAFE_TOBREAK or when
```

Comment:
Is it really called UNSAFE_TOBREAK, or should that be UNSAFE_TO_BREAK ?

Reply:
Fixed. Thanks.